doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -632,6 +632,24 @@ In the Fast Pair not discoverable advertising mode, the Provider informs the lis
 You can also use the :c:func:`bt_fast_pair_has_account_key` function to check whether your Provider has any Account Keys.
 This API is especially useful after a system reboot when some Account Keys may already be stored in non-volatile memory.
 
+.. _ug_bt_fast_pair_gatt_service_bond_management:
+
+Fast Pair bond management functionality
+=======================================
+
+To enable the Fast Pair bond management functionality, use the :kconfig:option:`CONFIG_BT_FAST_PAIR_BOND_MANAGER` Kconfig option.
+When this functionality is enabled, the Fast Pair subsystem tracks the Bluetooth bonds created through the Fast Pair Procedure and unpairs them if the procedure is incomplete or the Account Key associated with the bonds is removed.
+It also unpairs the Fast Pair Bluetooth bonds on Fast Pair factory reset, because the factory reset removes all Account Keys stored on device.
+See the :kconfig:option:`CONFIG_BT_FAST_PAIR_BOND_MANAGER` Kconfig option help for more datails about using the functionality.
+
+The Fast Pair bond management functionality is disabled by default.
+It is highly recommended to enable it for the following use cases of the Google Fast Pair application:
```

Comment:
The listed use-cases imply bond management. Can we rephrase a bit to clearly indicate that (current wording suggests that user needs to enable it manually)?

Reply:
There's a difference between the use case and the kconfig that enables the use case. User should see the kconfig to find the required feature set.

Reply:
From the use case section: `The selected Kconfig option configures the Fast Pair features and extensions to satisfy the `Fast Pair Device Feature Requirements`_ for your target use case.`

Reply:
I'd rather leave it as is. I don't want to introduce the use case Kconfigs at this point of documentation. They are introduced later.

Reply:
Discussed offline. We will use `Make sure` syntax here.